compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/scene/ComposeSceneMediator.desktop.kt

```diff
@@ -439,7 +441,10 @@ internal class ComposeSceneMediator(
         val size = sceneBoundsInPx?.size ?: container.sizeInPx
         val boundsInWindow = IntRect(
             offset = offsetInWindow,
-            size = size
+            size = IntSize(
```

Comment:
Is the issue due `container.sizeInPx` being negative?

Reply:
Yep. I've put it here to apply this logic to `sceneBoundsInPx` too (just in case).